shelley/chain-and-ledger/executable-spec/src/Cardano/Ledger/Core.hs

```diff
@@ -44,6 +55,30 @@ type family Script era :: Type
 -- | AuxiliaryData which may be attached to a transaction
 type family AuxiliaryData era :: Type
 
+-- | Transition system that processes the update payload for the era.
+type family UpdateSTS era :: Type
+
+class HasUpdateLogic era where
+  initialUpdateState :: State (UpdateSTS era)
```

Comment:
:warning: In the Priviledge prototype we need to provide the "genesis protocol".

Reply:
Can we remove the initial rules?